src/Protocol/Server/Capabilities/TextDocumentSyncOptions.cs

```diff
@@ -38,17 +39,24 @@ public class TextDocumentSyncOptions : ITextDocumentSyncOptions
 
         public static TextDocumentSyncOptions Of(IEnumerable<ITextDocumentSyncOptions> options)
         {
-            return new TextDocumentSyncOptions() {
-                OpenClose = options.Any(z => z.OpenClose),
-                Change = options
+            var change = TextDocumentSyncKind.None;
+            if (options.Any())
+            {
+                change = options
                         .Where(x => x.Change != TextDocumentSyncKind.None)
-                        .Min(z => z.Change),
+                        .Min(z => z.Change);
```

Comment:
As per comment in https://github.com/OmniSharp/csharp-language-server-protocol/issues/162#issuecomment-519433579 this seems to throw because sequence doesn't contain any elements. Maybe add the same predicate `x => x.Change != TextDocumentSyncKind.None` also to the `if (options.Any())` check above?

Reply:
sounds like a good change...